src/Admin/Model/UserAdmin.php

```diff
@@ -20,13 +20,30 @@
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Show\ShowMapper;
 use Sonata\UserBundle\Form\Type\SecurityRolesType;
+use Sonata\UserBundle\Model\UserManagerInterface;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 /**
  * @phpstan-extends AbstractAdmin<\Sonata\UserBundle\Model\UserInterface>
  */
 class UserAdmin extends AbstractAdmin
 {
+    private ?UserManagerInterface $userManager;
+
+    public function setUserManager(UserManagerInterface $userManager): void
```

Comment:
Wouldn't it better to use constructor ?

Reply:
I set the min version to 4.8 of SonataAdmin and use the constructor.